Resources/public/js/models/ez-contentmodel.js

```diff
@@ -464,6 +464,26 @@ YUI.add('ez-contentmodel', function (Y) {
             }
             contentService.deleteContent(this.get('id'), callback);
         },
+
+        /**
+         * Removes translation
+         *
+         * @method removeTranslation
+         * @param {Object} options
+         * @param {Object} options.api (required) the JS REST client instance
+         * @param {String} languageCode
+         * @param {Function} callback
+         */
+        removeTranslation: function (options, languageCode, callback) {
+            var capi = options.api,
+                contentService = capi.getContentService();
+
+            if ( !this.get('id') ) {
+                return callback(false);
```

Comment:
This will be displayed as a success, I'm not sure if we want this.

Reply:
True. There is no checking of id in other methods (besides `delete`) so I've removed this condition (API should return 404 in this case).